src/Stepping.TmProviders.LocalTm/Stepping/TmProviders/LocalTm/DistributedLocks/DefaultSteppingDistributedLock.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Concurrent;
+
+namespace Stepping.TmProviders.LocalTm.DistributedLocks;
+
+public class DefaultSteppingDistributedLock : ISteppingDistributedLock
```

Comment:
Maybe the local lock is useless?

Reply:
Default local lock is helpful in a single instance environment.

Reply:
I notice that the LocalTmHostedService is a singleton service that invokes ProcessPendingAsync one at a time.

https://github.com/TeamStepping/Stepping.NET/blob/6c62c612a58f9961157d0ec12c666ce63547b880/src/Stepping.TmProviders.LocalTm/Stepping/TmProviders/LocalTm/HostedService/LocalTmHostedService.cs#L38